CHANGELOG.md

```diff
@@ -51,6 +51,12 @@ Our versioning strategy is as follows:
 
 ### 🧹 Chores
 
+## 22.1.1
+
+### 🎉 New Features & Improvements
```

Comment:
Under the features and improvements I would add a note regarding "GraphQLRequestClient" that the "request" method supports dynamic headers